src/SFA.DAS.Forecasting.Web/Orchestrators/ForcastingOrchestrator.cs

```diff
@@ -89,7 +89,7 @@ ICommitmentsDataService commitmentsDataService
             var result = await _accountProjection.Get(accountId);
             var d = _mapper.MapProjections(result);
 
-            return d.Where(m => m.Date.IsAfterOrSameMonth(DateTime.Today) && (!_applicationConfiguration.LimitForecast || m.Date < BalanceMaxDate))
+            return d.Where(m => m.Date.IsAfterOrSameMonth(DateTime.Today.AddMonths(1)) && (!_applicationConfiguration.LimitForecast || m.Date < BalanceMaxDate))
                 .OrderBy(m => m.Date)
                 .Take(48)
```

Comment:
Should we change this to be 49 - to incorporate the fact that we are skipping one?

Reply:
Don't think so. 
We only want 48 months anyway. 
The reason we added take(48) was that we changed to projection function to generate 49 months (still does).
/c